### DISCUSSION TITLE
Meta title is not updated in browser tab on route change.

### DISCUSSION THREAD

**Discussion by darrylmorley** [vercel/next.js#52332] [Help] [answered]
### Summary

Only the initial selected route title is shown in the browser tab. When following links the title in the browser tab does not change, despite the page title changing in the source.

### Additional information

```js
The routes that don't update are children of a Radix UI NavigationMenu component and use next/link.
```

edit: I should add that this is on ISR pages only

**Comment by icyJoseph** [comment]
How are you updating the document title?

**Reply by darrylmorley** [reply]
Like this: 
```
export const metadata = {
  title: 'my title'
  ...
}
```

**Comment by darrylmorley** [comment, marked as answer]
Fixed in v13.4.9

https://github.com/vercel/next.js/issues/49409
